waterlog/src/Components/Segments.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegments } from '../actions/SegmentActions';
+
+class Segments extends Component {
+    componentWillMount() {
+        this.props.fetchSegments();
+      }
+  render() {
+    const segments = this.props.segments.map(segments => (
+        <div key={segments.id}> 
+           <p>{"id: "+segments.id}</p>
+           <p>{"sense ID Out: "+segments.senseIDOut}</p>
+           <p>{"sense ID In: "+segments.senseIDIn}</p>
```

Comment:
Check your capitalisation of displayed text:
`"id: "+` probably should be `"Id: "` 
`sense ID Out: "` should be `Sense ID out:` etc.....